public/docs/ts/latest/cookbook/i18n.jade

```diff
@@ -179,23 +183,24 @@ a#cardinality
   :marked
     This syntax conforms to the
     <a href="http://userguide.icu-project.org/formatparse/messages" target="_blank" title="ICU Message Format">ICU Message Format</a>
-    which derives from the 
-    <a href="http://cldr.unicode.org/" target="_blank" title="CLDR">Common Locale Data Repository (CLDR)</a>
+    that derives from the 
+    <a href="http://cldr.unicode.org/" target="_blank" title="CLDR">Common Locale Data Repository (CLDR),</a>
     which specifies the
     <a href="http://unicode.org/reports/tr35/tr35-numbers.html#Language_Plural_Rules" target="_blank" title="Pluralization Rules">pluralization rules</a>.
 
 a#select
 :marked
-  ## Select
+  ## Select among alternative texts
   The application displays different text depending upon whether the hero is male or female.
-  These text alternatives require translation too.
+  These text alternatives require translation, too.
```

Comment:
My mistake. I thought that was the rule but on looking it up it looks like your way is the right way. Removed.